daily-challenges/2024-12/11/ducnq.k11.spec.ts

```
@@ -0,0 +1,24 @@
+
+let menhGia = [500000, 200000, 100000, 50000, 20000, 10000, 5000, 2000, 1000];
+let result = {};
+
+function moneyExchange(amount:any){
+    if(amount < 1000){
+        return null;
+    }
+    for(let i = 0; i < menhGia.length; i++){
+        let soLuong = Math.floor(amount/menhGia[i]);
+        if(soLuong > 0){
+            result[menhGia[i]] = soLuong;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'soLuong' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)